src/tokens/transformation/tailwind/twFont.ts

```diff
@@ -1,17 +1,18 @@
 import { Transform } from 'style-dictionary'
+import { fontFamily } from '../common/fontHelper'
 
 export default {
   type: 'value',
   matcher(token) {
     return token.type === 'custom-fontStyle'
   },
-  transformer({ value: font }) {
+  transformer({ value: font }, platform) {
     return {
       fontSize: `${font.fontSize}px`,
       lineHeight: `${font.lineHeight}px`,
       letterSpacing: `${font.letterSpacing}px`,
       fontWeight: font.fontWeight,
-      fontFamily: font.fontFamily
+      fontFamily: fontFamily(font, platform?.options as any)
```

Comment:
do we actually pass anything through has `fontFamilies`?

Reply:
Not sure, but seems like it's still worth handling since it's theoretically possible?